woocommerce/woocommerce-gutenberg-products-block
# Refactor frontend, frontend-container-block (of reviews) to TypeScript

## What

Fixes #9527

## Why

We are converting JS components to TypeScript for consistency.

## Testing Instructions

This component is part of the reviews block so the approach I took was to test this block and that it renders and functions correctly in front-end pages.

Try out the various review blocks in editor, then check they render correctly in the front-end. Also check that the order by dropdown works as expected.



## Screenshots or screencast

n/a

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
